pht_federated/client/storage/fs.py

```diff
@@ -7,7 +7,7 @@
 
 class MinioBuckets(enum.Enum):
     MODELS = 'models'
-    DATASETS = 'datasets'
+    DATASETS = 'discoveries'
```

Comment:
why is there a discoveries bucket?

Reply:
hmm hab ich ziemlich sicher nicht erstellt